resources/views/admin/user-list.blade.php

```diff
@@ -0,0 +1,77 @@
+@extends('layouts.master')
+
+@section('content')
+
+@include('layouts.sidebar')
+
+@include('layouts.main-content')
+
+@php
+    use App\Models\User;
+@endphp
+
+<div class="list">
+    <div class="list-header">
+        <div>
+            <span>{{ __('message.ul-total') }}: </span><span>{{ $users->count() }}</span>
+        </div>
+        <div class="list-header-search">
+            <form action="{{ route('user-list') }}" method="GET">
+                <input type="text" name="query" value="{{ old('query', request()->input('query')) }}" placeholder="Search user">
+                <button type="submit">
+                    <span><i class="fa-solid fa-magnifying-glass"></i></span>
+                </button>
+            </form>
+        </div>
+    </div>
+    <table class="table">
+        <thead>
+            <tr>
+                <th scope="col">#</th>
+                <th scope="col">{{ __('message.ul-name') }}</th>
+                <th scope="col">{{ __('message.ul-avatar') }}</th>
+                <th scope="col">{{ __('message.ul-email') }}</th>
+                <th scope="col">{{ __('message.ul-phone') }}</th>
+                <th scope="col">{{ __('message.ul-status') }}</th>
+            </tr>
+        </thead>
+        <tbody>
+            @foreach ($users as $key => $user)
+            <form action="{{ route('toggle-block', $user->id) }}" method="POST">
+            @csrf
+            @method('PUT')
+            <tr>
+                <th scope="row">{{ $key+1 }}</th>
+                <td>{{ $user->name }}</td>
+                <td>
+                    @if ($user->avatar)
+                    <img src="{{ Vite::asset('public/storage/upload/' . $user->avatar) }}" alt="">
+                    @endif
+                </td>
+                <td>{{ $user->email }}</td>
+                <td>{{ $user->phone }}</td>
+                <td>
+                    @foreach (User::STATUS as $key => $status)
+                    {{ $user->status == $key ? $status : '' }}
```

Comment:
sao lại là key?

Reply:
const STATUS = [
        0 => 'Not verified',
        1 => 'Block',
        2 => 'Verified',
];
trong constant em đang để giá trị key bằng với các giá trị trong cột status, nên em dùng để so sánh luôn

Reply:
chỗ này cứ in ra thôi for làm gì

Reply:
nếu in ra bình thường thì status sẽ là 0 1 2, em đang muốn hiển thị trạng thái của nó là block, verify
nếu không for thì em sẽ phải check từng cái một, hôm trước anh bảo phải chuẩn bị cho tất cả trường hợp nên em dùng for, em chưa tìm được cách nào hay hơn